man/coap_pdu_setup.txt.in

```diff
@@ -239,6 +263,9 @@ received a part of the data and request the next block (with the appropriate
 Block options) from the server.  Returning the next requested block is handled
 by this function.
 
+*NOTE:* This function has been superceeded by *coap_add_data_response_large*().
```

Comment:
According to [MW](https://www.merriam-webster.com/dictionary/supercede) "superseded" seems to be the preferred spelling.

Reply:
Corrected

Reply:
Hm, my spell checker thinks "superseded" rather than "superseeded". But this may be BE vs. AE. Feel free to leave it as is if you think "ee" is correct.

Reply:
Now superseded